src/ImageSharp/Formats/Jpg/JpegDecoderCore.cs

```diff
@@ -559,42 +547,42 @@ private void ProcessDefineHuffmanTablesMarker(int remaining)
             }
         }
 
-        private void ProcessDefineHuffmanTablesMarkerLoop(ref Huffman huffman, ref int remaining)
+        private void ProcessDefineHuffmanTablesMarkerLoop(ref HuffmanTree huffmanTree, ref int remaining)
```

Comment:
@antonfirsov I think the confusion is there as you are passing the int params by ref also. The compiler is heavily optimised for passing small structs so it's uncommon to see them passed by reference.